src/components/AboutUs/Subsidiaries.js

```diff
@@ -16,7 +16,7 @@ const Subsidiary = ({ image, description, linkUrl, linkText }) => (
         {description}
       </BodyPrimary>
       {linkText ? (
-        <StyledLink reverse href={linkUrl}>
+        <StyledLink reverse href={linkUrl} target="_blank">
```

Comment:
Our `StyledLink` component takes a prop called `external`, which adds `target="_blank"` but also `rel='noopener noreferrer'`
https://github.com/yldio/yld.io/blob/3ec6b582c7f8c3735559674d15284b867e5b745f/src/components/Common/StyledLink.js#L87-L107

([This site](https://support.performancefoundry.com/article/186-noopener-noreferrer-on-my-links) offers a good explanation of what noopener & noreferrer are all about)

So could we replace these `target="_blank"` with `external` :pray: ?

---

@Daveawb @antoniocapelo As a slight aside, we could make it easier for new developers to use our existing library of components, if we had more thorough documentation (i.e. storybook)? For this situation in particular, we could document the `external` prop more effectively, using the [links addon ](https://github.com/storybooks/storybook/tree/next/addons/links) & the [info addon](https://github.com/storybooks/storybook/tree/next/addons/info)?

Given the rotating nature of the staffing on this project, I would suggest that we start looking into a more comprehensive way of documenting our components than just the basic storybook functionality (e.g. additional info, like this). A combination of "info" and ["knobs"](https://github.com/storybooks/storybook/tree/next/addons/knobs) would be a good start in enabling us to explain/demonstrate more to devs.

If anyone has the time, [here's an article on using React + storybook in the context of a design system](https://medium.com/buildit/using-react-within-a-design-system-73d4bb0cc822)

Reply:
Agreed. New comers to the site need to know what is available.

Reply:
Yes, this documentation is absolutely necessary.

Reply:
Do we have a Trello ticket where we could add these notes? We should create one.